Library/FormatTests.swift

```diff
@@ -94,6 +94,44 @@ final class FormatTests: TestCase {
     }
   }
 
+  func testDecimalCurrency() {
+    XCTAssertEqual(Format.decimalCurrency(for: 10), "10.00")
+    XCTAssertEqual(Format.decimalCurrency(for: 10.00), "10.00")
+    XCTAssertEqual(Format.decimalCurrency(for: 10.50), "10.50")
+    XCTAssertEqual(Format.decimalCurrency(for: 10.5555), "10.55", "Rounds down to 2 fraction digits")
+    XCTAssertEqual(Format.decimalCurrency(for: 10.511), "10.51", "Rounds down to 2 fraction digits")
+  }
+
+  func testAttributedCurrency() {
+    withEnvironment(locale: Locale(identifier: "en")) {
+      withEnvironment(countryCode: "US") {
+        XCTAssertEqual(Format.attributedCurrency(1_000, country: .us)!.string, "$1,000.00")
```

Comment:
Again, I would suggest not using force unwrap here so `XCTAssertTrue` API might be considered as an alternative.

```swift
XCTAssertTrue(Format.attributedCurrency(1_000, country: .us)?.string == "$1,000.00")
...
```